Gemfile

```diff
@@ -6,14 +6,13 @@ source 'https://gem.fury.io/invoca'
 gemspec
 
 gem 'aggregate', source: 'https://gem.fury.io/invoca'
-
-group :development do
-  gem 'bundler', '~> 1.16'
-  gem 'elasticsearch-extensions'
-  gem 'pry'
-  gem 'pry-byebug'
-  gem 'rake', '~> 10.0'
-  gem 'rspec', '~> 3.0'
-  gem 'rubocop', '0.74.0', require: false
-  gem 'rubocop-git', require: false
-end
+gem 'appraisal'
+gem 'bundler', '~> 1.16'
+gem 'elasticsearch-extensions'
+gem 'pry'
+gem 'pry-byebug'
+gem 'rake', '~> 10.0'
+gem 'rspec', '~> 3.0'
+gem "rspec_junit_formatter", "~> 0.4"
+gem 'rubocop', '0.74.0', require: false
+gem 'rubocop-git', require: false
```

Comment:
Can you line up the versions in one column like you did in the gemspec? I find that a lot easier.